target/linux/ramips/image/mt76x8.mk

```diff
@@ -57,6 +61,19 @@ define Device/hc5661a
 endef
 TARGET_DEVICES += hc5661a
 
+define Device/ki_rb
+  DTS := ki_rb
+  IMAGE_SIZE := $(ralink_default_fw_size_16M)
+  UIMAGE_NAME := ZyXEL Keenetic Extra II
+  DEVICE_TITLE := ZyXEL Keenetic Extra II
+  SUPPORTED_DEVICES := zyxel,keenetic-extra-ii
```

Comment:
Due to the correct device define, the `SUPPORTED_DEVICES` can be dropped.

Reply:
ok